CONTRIBUTORS.yaml

```diff
@@ -2767,11 +2767,14 @@ tbrown91:
     orcid: 0000-0001-8293-4816
     joined: 2024-08
 
+tStehling:
```

Comment:
Please feel free to add more information, like your affiliation UFZ or an ORCID ID.